.travis.yml

```diff
@@ -1,5 +1,5 @@
 # File managed by pluginsync
-sudo: false
+sudo: true
```

Comment:
@rashmigottipati, Let's update .travis.yml same as the head version as it served its purpose.